### DISCUSSION TITLE
DIA-NN returning empty columns in the report

### DISCUSSION THREAD

**Discussion by aleksandrabinek** [vdemichev/DiaNN#723] [Q&A]
I’ve been using DIA-NN with PTMs heavy library searches. In this particular set of files only 12/61 files reported values in columns for the remaining 39 files the columns were just empty. I tried declaring the modifications in the dialog box (--mod) to see if helps and tried setting library generation strategy to "IDs, RT and IM profiling", but still the same 39 files returned empty columns in the report. I didn't have any issues with those files when using other software.
Please see the log attached. Would you be able to help figuring out why this occurred?
Thank you
Aleksandra
[report.log.txt](https://github.com/vdemichev/DiaNN/files/11710066/report.log.txt)

**Comment by vdemichev** [comment]
Hi Aleksandra,

Files failed to load, basically potentially incorrect settings used to convert to .mzML. DIA-NN can read the native file format, so no need to convert. Or these can be MSX-DIA files, which are not supported.

Best,
Vadim
